tests/test_balanced.py

```diff
@@ -3,7 +3,6 @@
 import json
 import numpy as np
 import pytest
-np.random.seed(0)
```

Comment:
Why did you remove this line?

Reply:
Please also look at the parent class TestMethods.fix_seed in tests/test_methods.py .
All the test classes for interleaving inherits this class and call TestMethods.fix_seed every time before a test method is called.
By doing this, we can fix the seed for all the test methods, which is independent from the execution order of test methods.

Reply:
I overlooked that. Thank you!